### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by eedwards-sk** [jpadilla/django-dotenv#42]
I looked at the source code for this project -- none of it appears to be django specific.

Why is it branded with django?

**Comment by woutervh** [comment]
I agree,  but there is another project called python-dotenv, but I always use django-dotenv.

both packages import as a dotenv-module,  so there is a naming conflict.
you cannot use both package in the same project

``
> pip install python-dotenv
> pip install django-dotenv

>>> import dotenv   # which one wins?
``

**Comment by Suor** [comment]
I hit this conflict too, trying to use both this and `pytest-dotenv`.

**Comment by skorokithakis** [comment]
Same here, this should have been named something else.

**Comment by aprilahijriyan** [comment]
Yes, I agree, the package name must be changed so that there is no conflict with `python-dotenv`. Maybe a good name is `django_dotenv` ?

**Comment by WillowP** [comment]
> I hit this conflict too, trying to use both this and `pytest-dotenv`.

Also hit this. I tried to fix it using [pytest-django-dotenv](https://github.com/kaniak274/pytest-django-dotenv), but they only support paths relative to the `VIRTUAL_ENV` environment var. I'm using `pyenv` so that doesn't cut it.

FWIW my solution was to switch from using the `pytest` command to using `python manage.py test`, enabled by the solution described in the docs [here](https://pytest-django.readthedocs.io/en/latest/faq.html#how-can-i-use-manage-py-test-with-pytest-django). I just copy/pasted the solution in the docs and this started working :)

Btw I'm using `pytest-django`

**Comment by suspiciousRaccoon** [comment]
```python-dotenv``` seems to work well enough for me. I use ```load_dotenv()``` in ```manage.py``` and I can access .env variables from settings. Does this project have any additional functionality?
